bundle/config/mutator/initialize_urls_test.go

```diff
@@ -84,17 +93,18 @@ func TestInitializeURLs(t *testing.T) {
 		"job1":             "https://mycompany.databricks.com/jobs/1?o=123456",
 		"pipeline1":        "https://mycompany.databricks.com/pipelines/3?o=123456",
 		"experiment1":      "https://mycompany.databricks.com/ml/experiments/4?o=123456",
-		"model1":           "https://mycompany.databricks.com/ml/models/model1?o=123456",
+		"model1":           "https://mycompany.databricks.com/ml/models/a%20model%20uses%20its%20name%20for%20identifier?o=123456",
```

Comment:
```suggestion
		"a model uses its name for identifier":           "https://mycompany.databricks.com/ml/models/a%20model%20uses%20its%20name%20for%20identifier?o=123456",
```

Reply:
`model1` is the resource key here, not the name.